ingestion/ingestion/ingest.py

```diff
@@ -122,19 +126,22 @@ def add_add_dataset_flags(parser: ArgumentParser):
     #         help=f"{kind} directory",
     #     )
 
-    def add_in_paths(parser: ArgumentParser, kind: str):
+    def add_in_paths(
+        parser: ArgumentParser, kind: str, *, short_flag: str | None = None
+    ):
         parser.add_argument(
-            f"-{kind.lower()[0]}",
+            "-" + (short_flag or f"{kind.lower()[0]}"),
```

Comment:
We can use full expressions in `{...}` in an f-string, so we can rewrite it directly:
`f"-{short_flag or kind.lower()[0]}"` which is a little bit easier to read :)